LICENSE.md

```diff
@@ -1,10 +1,10 @@
 # LICENSE
 
-Copyright © 2017 Bill & Melinda Gates Foundation
+Copyright © 2020 Mojaloop Foundation
```

Comment:
I would drop this change if it isn't coming from the `master`

Reply:
It is. Lewis seems to updated the licenses accross many of the repos.